.github/workflows/scan-code-trivy.yaml

```diff
@@ -0,0 +1,41 @@
+name: Trivy Scan Code
+
+on:
+  push:
+    branches: [main, develop]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main, develop]
+
+concurrency:
+  group: callee-trivy-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trivy-scan-code:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Cache Scan Dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/trivy
+          key: callee-trivy-${{ github.workflow }}-${{ github.run_id }}
+          restore-keys: trivy-
+      - name: Run Trivy vulnerability scanner in repo mode
```

Comment:
We might want to scan the container too, once it's available. I'll write a card for it